src/components/TaskBoard/TaskBoard.module.scss

```diff
@@ -0,0 +1,41 @@
+.board {
+    display: flex;
+    flex-direction: column;
+    width: 623px;
```

Comment:
usually such hardcoding height/width is a bad practice, it will not adjust to smaller/bigger screens